xenonpy/descriptor/fingerprint.py

```diff
@@ -672,12 +703,21 @@ def __init__(self, n_jobs=-1,
             Specify a single column to be used for transformation.
             If ``None``, all columns of the pd.DataFrame is used.
             Default is None.
+        desc_list: string or list
+            List of descriptor names to be called in rdkit to calculate molecule descriptors.
+            If ``classic``, the full list of rdkit v.2020.03.xx is used. (length = 200)
+            Default is to use the latest list available in the rdkit. (length = 208 in rdkit v.2020.09.xx)
         """
         # self.arg = arg # arg[0] = radius, arg[1] = bit length
         super().__init__(n_jobs=n_jobs, on_errors=on_errors, return_type=return_type, target_col=target_col)
         self.input_type = input_type
-        nms = [x[0] for x in Descriptors._descList]
```

Comment:
A little suggestion. Rename `Descriptors` to something like `ChemDescriptors` will be better for reading.
Modify `from rdkit.chem import Descriptors` to `from rdkit.chem import Descriptors as ChemDescriptors` will do this work.

Reply:
updated as you recommended to this branch